site/docs/.vuepress/styles/index.scss

```diff
@@ -6,9 +6,52 @@
 }
 
 html.dark {
+  --c-bg-lighter: #2b313a;
```

Comment:
Hmm, why did you make lighter the same color as light?
<img width="768" alt="image" src="https://user-images.githubusercontent.com/21295738/187081093-ff1b4c20-c1b3-4b9e-892c-5582efaba00c.png">
This makes code snippets inside tables blend in

Reply:
Oops... My bad.